packages/js/product-editor/src/components/variations-table/variation-actions-menus/variation-quick-update-menu-item.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import { Slot, Fill, MenuItem } from '@wordpress/components';
+import { createElement, Fragment } from '@wordpress/element';
+import {
+	createOrderedChildren,
+	sortFillsByOrder,
+} from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import { MenuItemProps } from './types';
+import { QUICK_UPDATE, SINGLE_UPDATE } from './constants';
```

Comment:
I think these constants should be renamed to `MULTIPLE_UPDATE` and `SINGLE_UPDATE`.

Reply:
I like it! [2a2b72a](https://github.com/woocommerce/woocommerce/pull/43441/commits/2a2b72ab663809c1e462229fe545d8cabddf46ab)